src/twilio/functions/sms/intake-delivery-txts.js

```diff
@@ -0,0 +1,99 @@
+exports.handler = function codeSms(context, event, callback) { // eslint-disable-line
+  const Airtable = require("airtable"); // eslint-disable-line
+  Airtable.configure({
+    endpointUrl: "https://api.airtable.com",
+    apiKey: context.AIRTABLE_API_KEY // eslint-disable-line
+  });
+
+  const base = Airtable.base("apppK7mrvMPcwtv6d"); // intake + vols base
+  let code = event.body.requestCode;
+  const deliveryPhone = event.body.deliveryPhone; // eslint-disable-line
```

Comment:
you can destructure each key like this:
```
let {
  deliveryPhone,
  deliveryName,
  intakePhone,
  intakeName } = event.body;
```
and now those variables will exist in scope and can be used. This should remove the eslint complaint as well.

Reply:
cool, made this change!